lib/mount.c

```diff
@@ -321,6 +321,65 @@ void fuse_kern_unmount(const char *mountpoint, int fd)
 	waitpid(pid, NULL, 0);
 }
 
+static int fuse_mount_fusermount_setup_auto_unmount_only(const char *mountpoint, int quiet)
```

Comment:
@Nikratio This is essentially a variant of `fuse_mount_fusermount`.

My first version crammed both variants into the same function.  That had less code duplication, but felt a bit too complicated.  Especially since the return value here communicates both errors xor a file desciptor.